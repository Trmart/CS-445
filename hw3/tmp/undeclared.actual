.   .   .   .   Child: 0  Id: i [line: 5]
.   .   .   Child: 0  Id: i [line: 5]
.   .   .   Child: 0  Id: z [line: 3]
.   .   .   Child: 0  Id: z [line: 4]
.   Child: 0  Parm: x of type int [line: 1]
.   .   Child: 1  Assign: <= [line: 3]
.   Child: 1  Compound [line: 2]
.   .   .   .   Child: 1  Const 3 [line: 5]
.   .   .   Child: 1  Id: i [line: 3]
.   .   .   Child: 1  Id: i [line: 4]
.   .   .   Child: 1  Op: + [line: 5]
Func: ( returns type void [line: 1]
.   .   Sibling: 1  Assign: <= [line: 4]
.   .   Sibling: 2  Assign: <= [line: 5]
