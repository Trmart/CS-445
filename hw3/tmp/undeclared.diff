1,14c1,14
< ERROR(3): Symbol 'i' is not declared.
< ERROR(3): Symbol 'z' is not declared.
< ERROR(4): Symbol 'i' is not declared.
< ERROR(4): Symbol 'z' is not declared.
< ERROR(5): Symbol 'i' is not declared.
< ERROR(5): Symbol 'i' is not declared.
< ERROR(6): Symbol 'j' is not declared.
< ERROR(7): Symbol 'l' is not declared.
< ERROR(8): Symbol 'm' is not declared.
< ERROR(9): Symbol 'k' is not declared.
< ERROR(LINKER): A function named 'main()' must be defined.
< Number of errors: 11
< Number of warnings: 1
< WARNING(1): The variable 'x' seems not to be used.
---
> .   .   .   .   Child: 0  Id: i [line: 5]
> .   .   .   Child: 0  Id: i [line: 5]
> .   .   .   Child: 0  Id: z [line: 3]
> .   .   .   Child: 0  Id: z [line: 4]
> .   Child: 0  Parm: x of type int [line: 1]
> .   .   Child: 1  Assign: <= [line: 3]
> .   Child: 1  Compound [line: 2]
> .   .   .   .   Child: 1  Const 3 [line: 5]
> .   .   .   Child: 1  Id: i [line: 3]
> .   .   .   Child: 1  Id: i [line: 4]
> .   .   .   Child: 1  Op: + [line: 5]
> Func: ( returns type void [line: 1]
> .   .   Sibling: 1  Assign: <= [line: 4]
> .   .   Sibling: 2  Assign: <= [line: 5]
