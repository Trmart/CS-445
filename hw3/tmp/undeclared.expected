ERROR(3): Symbol 'i' is not declared.
ERROR(3): Symbol 'z' is not declared.
ERROR(4): Symbol 'i' is not declared.
ERROR(4): Symbol 'z' is not declared.
ERROR(5): Symbol 'i' is not declared.
ERROR(5): Symbol 'i' is not declared.
ERROR(6): Symbol 'j' is not declared.
ERROR(7): Symbol 'l' is not declared.
ERROR(8): Symbol 'm' is not declared.
ERROR(9): Symbol 'k' is not declared.
ERROR(LINKER): A function named 'main()' must be defined.
Number of errors: 11
Number of warnings: 1
WARNING(1): The variable 'x' seems not to be used.
